packages/rainbowkit/src/components/RainbowKitProvider/RainbowKitChainContext.tsx

```diff
@@ -1,29 +1,27 @@
 import React, { ReactNode, createContext, useContext, useMemo } from 'react';
-import { Chain as WagmiChain } from 'wagmi';
+import type { Chain } from 'wagmi/chains';
+import { mainnet } from 'wagmi/chains';
 import { provideRainbowKitChains } from './provideRainbowKitChains';
 
-export interface RainbowKitChain {
+export interface RainbowKitChain extends Chain {
   id: number;
-  name?: string;
   iconUrl?: string | (() => Promise<string>) | null;
   iconBackground?: string;
 }
 
-// This type is a combination of wagmi and RainbowKit chain types to make
-// it easier for consumers to define their chain config in a single place.
-export type Chain = WagmiChain & RainbowKitChain;
```

Comment:
Made some cleanups here for better types. Looks better if we're extending types rather than creating another `Chain` type here